kentico-mvc-environment/azuredeploy.json

```diff
@@ -0,0 +1,319 @@
+      }
+    },
+    "sqlServerName": {
+      "type": "string",
+      "metadata": {
+        "description": "Server name of the deployed Azure SQL server. The server name can contain only lowercase letters, numbers, and '-', but can't start or end with '-' or have more than 63 characters. The deployed server will be available at SQLServerName.database.windows.net. Keep in mind that the server name must be unique across the entire Azure environment, not only within your hosting plan."
+      }
+    },
+    "sqlAdministratorLogin": {
+      "type": "string",
+      "metadata": {
+        "description": "Login for the Azure SQL server administrator. "
+      }
+    },
+    "sqlAdministratorPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Azure SQL server administrator account. The password is case-sensitive and may contain lower case, upper case, numbers and special characters. The default Azure password complexity rules: minimum length of 8 characters, minimum of 1 uppercase character, minimum of 1 lowercase character, minimum of 1 number. Moreover, the password cannot contain the account name of the user account performing the deployment. The password can be up to 128 characters long."
+      }
+    },
+    "connectionStringDatabaseName": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Database name substituted in the 'Initial Catalog' parameter of the administration and website Web App connection strings. If left empty, the Web Apps are deployed without preconfigured connection strings."
+      }
+    },
+    "location": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "resources": [
+    {
+      "apiVersion": "2018-05-01",
+      "name": "pid-72505988-e227-4ca8-83ce-f2725a353ab4",
+      "type": "Microsoft.Resources/deployments",
+      "properties": {
+        "mode": "Incremental",
+        "template": {
+          "$schema": "https://schema.management.azure.com/schemas/2018-05-01/Microsoft.Resources.json",
+          "contentVersion": "1.0.0.0",
+          "resources": []
+        }
+      }
+    },
+    {
+      "name": "[parameters('sqlserverName')]",
+      "type": "Microsoft.Sql/servers",
+      "location": "[parameters('location')]",
+      "tags": {
+        "displayName": "SqlServer"
+      },
+      "apiVersion": "2018-06-01-preview",
+      "properties": {
+        "administratorLogin": "[parameters('sqlAdministratorLogin')]",
+        "administratorLoginPassword": "[parameters('sqlAdministratorPassword')]"
+      },
+      "resources": [
+        {
+          "type": "firewallrules",
+          "apiVersion": "2015-05-01-preview",
```

Comment:
update apiVersion

Reply:
According to the documentation, there is no newer version of the API. https://docs.microsoft.com/en-us/azure/templates/microsoft.sql/2015-05-01-preview/servers/firewallrules

Reply:
use the same apiVersion as the parent resource

Reply:
used the same apiVersion as parent